dali/test/python/test_operator_color_twist.py

```diff
@@ -81,9 +80,9 @@ def ref_color_twist(img, H, S, brightness, contrast, out_dtype):
     s, c = math.sin(angle), math.cos(angle)
     # Rotate the color components by angle and scale by S.
     # The fun part is that it doesn't really matter that much which
-    hmat = np.array([[1,    0,    0],
-                     [0,  c * S,  s * S],
-                     [0, -s * S,  c * S]])
```

Comment:
Was that actually wrong (i.e. triggering the linter?). In my PR I am applying kind of opposite review remarks: it seems we are going to edit such lines of code back and forth: from aligned columns to single whitespace.

Reply:
Must've happened by mistake. Reverted